### DISCUSSION TITLE
Visual Basic .NET support

### DISCUSSION THREAD

**Discussion by ashrafulislambd** [nxrighthere/UnrealCLR#70] [General]
Visual Basic .NET was the first and primary language for .NET CLR. But I can't see it in Unreal CLR. I only see C# and F#. Visual Basic should be supported. It would be of very use.

**Comment by jeswin** [comment]
Visual Basic is essentially in maintenance mode according to Microsoft. https://visualstudiomagazine.com/articles/2020/03/12/vb-in-net-5.aspx

If you're a VB programmer, give C# a try.

**Comment by nxrighthere** [comment]
You can use Visual Basic if you want. In general, any language that compiled into .NET assemblies will work.
